src/assets.ts

```diff
@@ -421,7 +421,7 @@ export function createAttachConfiguration(): string {
         "name": ".NET Core Attach",
         "type": "coreclr",
         "request": "attach",
-        "processId": "\${command:pickProcess}"
+        "processId": ""
```

Comment:
>"processId": "" [](http://example.com/codeflow?start=8&length=15)

What do you think about removing it entirely instead?

Reply:
I also had the same thought, if users wanted to override it themselves then they would most likely investigate the options in `launch.json` and set it themselves.

Reply:
Sure, but that seems rare, and if you want to do it, seems like `processName` would make more sense than `processId`. The only reason I can think to of using `processId` is if you are on an image without `ps` (or with a `ps` that acts differently than we expect).